### ISSUE TITLE
Sys-admin log

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1671]
We've discussed a number of times ways to make sys-admin work more visible. We've not yet found the "right" solution, but this is standing issue where summaries of sys-admin work can be put weekly. A [sys-admin](https://user-images.githubusercontent.com/4953703/93793648-fce85080-fc2e-11ea-824c-59fffd76c9ce.png) log of sorts.

**Comment by JosephMcArthur** [comment]
@markmacgillivray anything to add here from the last couple of weeks (ignoring the CF exploration).

**Comment by markmacgillivray** [comment]
Not much, the last couple of weeks has all overlapped with CF work or with the alert tuning.  I did review some changes made by Digital Ocean to their pricing plans which affect the resources we can deploy, but none of it offers us potential to save money just now. It does annoyingly make our recovery strategy slightly more difficult because they don't offer VMs in the same size as the one our ES cluster machines use any more, so any total failure would require me to make a new machine image. That's not too hard though, would take about an extra hour or two, and is very unlikely to happen - DO haven't failed at all since we started using them, something like five years ago. And once we eventually move to CF and upgrade our ES machine, we'll be on the newer Digital Ocean VM sizes then anyway.

DO also had some network maintenance announcements which I checked, none of them clash with our services and none of them need me to monitor anything, so we are OK.

Codeship had a bit of an issue two or three weeks ago as you know, and I've been re-setting our deployment stuff, which then oddly disappeared from codeship. That didn't take long to look at, but it was a little bit of extra work that I did. 

Ha I should definitely remember to write these things in the issue as I do them! There are probably more I've forgotten.
